docs/fastdds/transport/announced_locators.rst

```diff
@@ -29,7 +29,12 @@ as follows:
 External Locators
 -----------------
 
-The user can configure a set of external locators for each of the lists of unicast locators.
+The user can configure a set of external locators for each of the lists of unicast locators:
+
+* |BuiltinAttributes::metatrafficMulticastLocatorList-qos-api| on :ref:`wireprotocolconfigqos`
```

Comment:
I do not think this is the correct reference. I am quite sure the one intended here is `|BuiltinAttributes::metatraffic_external_unicast_locators-qos-api|`